cavern/README.md

```diff
@@ -57,6 +57,15 @@ org.opencadc.cavern.resourceID = ivo://{authority}/{name}
 # (optional) identify which container nodes are allocations
 org.opencadc.cavern.allocationParent = {top level node}
 
+# (optional) API Keys with administrative privileges.  Multiple keys can be specified, one per line.
+# Example:
```

Comment:
I prefer this list of config keys to be pretty minimal (no examples) and then explained in the paragraphs below. An example down with the explanation is fine if it helps.

Also, this is missing the `defaultQuotaGB` which should be optional and listed after the quota plugin (for context).